src/projects/pull-request/pr-status.html

```diff
@@ -76,7 +76,8 @@
           }
         },
         status: {
-          type: String
+          type: String,
+          notify: true
```

Comment:
To check for `_PrStatus` at https://github.com/preview-code/rite-evaluation/pull/313/files/88a56187b56228713f5c62bf09e7d6e1bd80ae5c#diff-6dfd009aa924dc588522f4da76e5a726R203